lib/espresso-runner.js

```diff
@@ -107,6 +135,20 @@ class EspressoRunner {
   }
 
   async buildNewModServer () {
+    let versions = {};
+    if (this.espressoBuildConfig) {
+      logger.info(`Using build configuration JSON from: '${this.espressoBuildConfig}'`);
+      versions = JSON.parse(await fs.readFile(this.espressoBuildConfig));
```

Comment:
there should be a proper human-readable message here if the file content is not a valid JSON

Reply:
readFile returns buffer by default. Provide the actual encoding as the second parameter to turn the result to a string